### PR TITLE
Add option to show password in password field

### PR DESCRIPTION
**Pull request by poncovka**
User is now able to display his user/admin password when creating it via clicking an icon.

### CODE REVIEW

File: data/pixmaps/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pixmaps/Makefile.am for anaconda
+#
+# Copyright (C) 2009  Red Hat, Inc.
```
Comment (jkonecny12) [inline data/pixmaps/Makefile.am]: Please change the year here and write yourself as an author of this file.

File: pyanaconda/ui/gui/__init__.py [hunk 2]
```diff
@@ -865,6 +865,11 @@ def run(self):
             Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(), provider,
                     Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION)
 
+            # Add the application icons to the theme
+            icon_path = os.environ.get("ANACONDA_DATA", "/usr/share/anaconda/pixmaps")
```
Comment (vpodzime) [inline pyanaconda/ui/gui/__init__.py]: I think this needs to be ``os.path.join(ANACONDA_DATA, "pixmaps")`` similarly to how ``css_path`` is constructed above.

File: pyanaconda/ui/gui/utils.py [hunk 3]
```diff
@@ -583,3 +585,20 @@ def unwatch_children(widget, callback, user_data=None):
...
+        text = PASSWORD_HIDE
+    else:
+        icon = PASSWORD_SHOW_ICON
+        text = PASSWORD_SHOW
+
+    entry.set_visibility(visible)
+    entry.set_icon_from_icon_name(position, icon)
+    entry.set_icon_tooltip_text(position, text)
+
+
```
Comment (vpodzime) [inline pyanaconda/ui/gui/utils.py]: A nitpick - one extra blank line at the end of the file ;-)

File: pyanaconda/ui/gui/__init__.py (hunk 2 above)
Comment (poncovka) [inline pyanaconda/ui/gui/__init__.py]: `css_path `is constructed as `css_path = os.environ.get("ANACONDA_DATA", "/usr/share/anaconda")`.

**Comment by poncovka** [comment]
Fixed requested changes and covered the disk encryption passphrase.

File: pyanaconda/ui/gui/__init__.py [hunk 4]
```diff
@@ -857,14 +857,21 @@ def run(self):
             settings = Gtk.Settings.get_default()
             settings.set_property("gtk-font-name", "Cantarell")
 
+            # Get the path to the application data
+            data_path = os.environ.get("ANACONDA_DATA", "/usr/share/anaconda")
+            
             # Apply the application stylesheet
-            css_path = os.environ.get("ANACONDA_DATA", "/usr/share/anaconda")
-            css_path = os.path.join(css_path, "anaconda-gtk.css")
+            css_path = os.path.join(data_path, "anaconda-gtk.css")
```
Comment (vpodzime) [inline pyanaconda/ui/gui/__init__.py]: Nice and elegant solution!

**Pull request merged**